README.md

```
@@ -1,88 +1,131 @@
 # Minifier Benchmarks (Beta)
 
-A collection of regularly updated benchmarks for HTML minifiers ([context](https://dev.to/j9t/on-the-need-for-neutral-maintained-minifier-metrics-5715)). Alpha release.
+A collection of regularly updated benchmarks for HTML minifiers ([context](https://dev.to/j9t/on-the-need-for-neutral-maintained-minifier-metrics-5715)).
 
-* **Minifier user?** Compare HTML minifiers [by HTML minification effectiveness](#html-minification-compared), [by overall “maximum” effectiveness](#maximum-minification-compared) (all available options checked), and the respective processing times
-* **Minifier author?** [Add and configure your minifier and become a co-owner](https://github.com/j9t/minifier-benchmarks/compare)
-* **Innocent bystander?** [Please share feedback and suggestions](https://github.com/j9t/minifier-benchmarks/issues/new)
+**⚠️ Early release, benchmarks are still being checked and fine-tuned.**
+
+* **👩‍💻 Minifier user?** Compare HTML minifiers [by HTML minification effectiveness](#html-minification-compared), [by overall “maximum” effectiveness](#maximum-minification-compared) (all available options checked), and by the respective processing times
```

markdownlint-cli2 (0.20.0)

[warning] Link fragments should be valid

(MD051, link-fragments)